db/migrate/20220810131220_add_automated_activity_to_audits.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class AddAutomatedActivityToAudits < ActiveRecord::Migration[6.1]
+  # rubocop:disable Metrics/MethodLength
+  def up
+    add_column(
+      :audits,
+      :automated_activity,
+      :boolean,
+      default: false,
+      null: false
```

Comment:
Does it make sense to have `null: false` when there's a default? I can't decide!

Reply:
There's this [article](1) which states why is best to have `null: false, default: false`

[1]:https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem

Reply:
Can you reshare? I think you've linked the wrong thing 😆

Reply:
That’s weird tried to edit the commit and the url is correct, here it is https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem

Reply:
Awesome, thank you!